ben-goldschmidt/25-loopsFor.js

```diff
@@ -0,0 +1,391 @@
+// Programming Basics: For Loops
+// These exercises are designed for practising "for" loops. Print all your results to the console.
+
+
+console.log("")
+console.log("QUESTION 1: ADDITION")
+// Addition. Write a program to add up the numbers 1 to 20.
+
+let addition1 = 20;
+let sum1 = 0;
+for (let i = 0; i <= addition1; i++) {
+    sum1 = sum1 + i;
+}
+console.log("The sum of 1 to 20 is", sum1);
+// The sum of 1 to 20 is 210
+
+
+
+console.log("")
+console.log("QUESTION 2: ADDITION and ARRAY")
+// There are i bottles of beer on the wall. Write a program that will output, "There is one bottle of beer on the wall." "There are two bottles of beer on the wall" up until there are five bottles.
+
+let bottles1 = 0;
+for (let i = 0; i < 5; i++) {
```

Comment:
we need if statement to manage
if one "There is one bottle of beer on the wall."
more than one  "There are two bottles of beer on the wall"